src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isNotUndefined } from 'ramda-adjunct';
+
+import { BannerProps } from './Banner.types';
+import { BannerVariants } from './Banner.enums';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import { Inline, Padbox } from '../layout';
+import colors from '../../theme/colors';
+import { IconTypes, SSCIconNames } from '../../theme/icons/icons.enums';
+import { getColor, pxToRem } from '../../utils';
+
+const colorVariants = {
+  [BannerVariants.info]: colors.radiantBlueberry,
```

Comment:
This isn't probably the correct way because if some user updates the theme colors in their application, this component will still use the original one, so I think you should put the color names (not the values) and then use the `getColor`.

Reply:
Good point, it should be tight to color name